### ISSUE TITLE
How should we nominate and recognize SME in the TEC?

### ISSUE THREAD

**Issue by tamarandom** [CommonsBuild/coordination#742]
## 👇🏽  What is this issue about? Give as much detail as you can. 
Nate raised this. I'm dropping it here so it doesn't get lost. ;) 

## ✨  What does success look like? (i.e., the criteria to meet to call this issue done)
1. There is clarity around the SME role and how it is recognized in the TEC
2.
3.

## 🧍🏻‍♀️🧍🏻‍♂️  Who should be part of the Advice Process? (People who should be consulted about this issue. Those who will have valuable input.)
@liviade1 @GriffGreen @JuankBell @tamarandom 

## 👀  Who should be in the loop about this issue?
@Stewards

**Comment by natenatesoo** [comment]
Thanks Tam!

**Comment by liviade1** [comment]
Good question. I've been giving them the tag on discord. I think it's ok to be quite informal. Maybe messaging them before would be a better practice.

**Comment by natenatesoo** [comment]
@liviade1 Yes! I saw the discord tag which is really nice.  I think formally recognizing SME's is really important (simply notifying individuals that we are classifying them as such).  Along with that, I think it would be extremely beneficial to "onboard" and "inform" SME's on their role, what to expect, and how to be effective.  By doing that, I think we can avoid potential conflict that may arise within advice processes.

**Comment by tamarandom** [comment]
This doc is somewhat out of date now but it does address SME recognition. 

[Onboarding and Resources Guide to the TEC](
https://docs.google.com/document/d/1UDnf26QQqqc77mnhn80MLCVzJU6IdvouT-OXPrIVXCg/edit#heading=h.tpx9g4b760ky)

CC: @danelsuga @liviade1

**Comment by natenatesoo** [comment]
Update: Still In Progress

**Comment by natenatesoo** [comment]
Update: SME's will be further detailed during this upcoming Sprint.  Working with Transparency and Soft-Gov as we fully develop the Advice Process.
